### ISSUE TITLE
AsieLauncher closes with Launching Minecraft

### ISSUE THREAD

**Issue by Slind14** [asiekierka/asielauncher#49]
Hey,

I followed the instructions step by step. And every time after he downloaded all depencies he says "Launching Minecraft", closes and nothing more happens. Is there any way to turn on a console or a hidden log I couldn´t find ?

**Comment by Slind14** [comment]
I did some more testing and know now that it works fine for most people. But it is not launching minecraft for me. It just closes the Asie launcher. I did a lot of fresh reinstalls and disabled the anti virus software and firewall. No chance.

Here is the log, but it says like nothing and looks the same for other people, where it works:
http://hastebin.com/tayahesuvu.tex
It doesn´t even come that far that it would create a forge log.

**Comment by Gamoholic** [comment]
Make sure you only have 64-bit Java installed. I know there are issues with having both 32 and 64 at the same time.

**Comment by Slind14** [comment]
that might solve the problem for me. But I need 32bit java for another app and I don´t like the idea of telling everyone to deinstall java 64bit who wants to use the launcher. Asie launcher should detect java and select a 32 bit version and if none found declare about it.

**Comment by Gamoholic** [comment]
Yes, it should. Asie knows about this, but has pretty much abandoned this project because FTB stole his thunder. Thank you for your interest in this project, but I really don't think much is going to happen now. Hopefully I am wrong.

**Comment by Slind14** [comment]
thunder ?

**Comment by Gamoholic** [comment]
It is an expression. FTB did what Asie wanted to do before he could get it done.

**Comment by Slind14** [comment]
well ftb is not making it public ? if you mean the sync of the launcher and server ? and I think sks launcher works the same and is a lot older.

**Comment by asiekierka** [comment]
SK's does not. MCUpdater is the closest, but its configuration system is an abomination to mankind.

The unique part was to be able to create a modpack by simply putting in the mods and configs and having the launcher do the rest for you. Curse and FTB are introducing that with their new launcher, so I decided bothering with a mod bazaar or any of the new features is pointless as Curse will be superior anyway - they have bags of money behind them.

About the bug... if I ever feel bored, I might fix it, but otherwise you're on your own. Sorry to the people who have expected this to become a huge project, especially ones who have donated in order to make this a huge project.

**Comment by Slind14** [comment]
You are aware that the curse client will take +6 month according to their statement and that people might want to have their own client which they can customize and not the curse client which will be strict.

**Comment by asiekierka** [comment]
The AsieLauncher does that. Most of the bugs don't stop you from using the launcher.

Also, if you really want help, upload the paste to somewhere else than hastebin. I couldn't even open it.

**Comment by Slind14** [comment]
I know, I just don´t understand why you give up your project if curse want to do it the same way. Its still a different target group. Well the log said nothing, so it doesn´t matter.

**Comment by jakimfett** [comment]
Fortunately, there are people in the community who are interested in seeing this continue to be developed, including myself. I'm finishing up a development project at the end of this month, after which there should be a fairly major update to a couple of things, starting with the server list and the update service.

**Comment by Gamoholic** [comment]
I just realized this might be the stupid Scala issue. Try adding this to AsieLauncher/lib on your AsieLauncher server. http://repo.maven.apache.org/maven2/org/scala-lang/scala-library/2.10.2/scala-library-2.10.2.jar
